.github/actions/setup-sql-server/Initialize-SqlServer.ps1

```diff
@@ -1,10 +1,11 @@
 param (
+    [string]$sqlServerExpressVersion = '2022.16.0.1000',
```

Comment:
https://community.chocolatey.org/packages/sql-server-express says "Requires a reboot due to dependencies." How come it doesn't?

Reply:
I didnt look into that but we used sql express before osoe 481. So it is working for a while. I will try to check that.

Reply:
![image](https://github.com/Lombiq/GitHub-Actions/assets/92299130/4ebc0cf0-e858-4f79-8e6e-4e801c36e19b)
![image](https://github.com/Lombiq/GitHub-Actions/assets/92299130/1d2ee135-7b55-4163-a76e-ea7f26ea386b)
According to this there is no need to reboot.

Reply:
It more visible here:
![image](https://github.com/Lombiq/GitHub-Actions/assets/92299130/e3f356aa-9bbb-434a-92b2-aa1a75a7dc0b)

Reply:
This seems conflicting for me. Anyway, let's leave it like that.